Automattic/jetpack
# Podcast Player: Fix focus styles

Fixes #15537

#### Changes proposed in this Pull Request:
- Make focus outline appear for ME.js buttons,
- Make box-shadow for focused anchors appear for themes that define it,
- Prevent the podcast title's outline from being cut-off.

#### Testing instructions:
- Add Player block (you can use this URL: https://distributed.blog/category/podcast/feed/),
- Focus the block and start tabbing through it - interactive elements should have some kind of outline (depends on current theme) when focused,
- Make sure the podcast title link is not cut-off,
- Test with a few themes in the editor and on the front-end.

#### Demo:

Theme: TwentyTwenty

![Apr-24-2020 15-10-42](https://user-images.githubusercontent.com/1451471/80216125-db624900-863d-11ea-879c-c31b0777a3c8.gif)


#### Proposed changelog entry for your changes:
* Podcast Player: Fix focus styles
